galasa-parent/dev.galasa.framework.api.authentication/src/main/java/dev/galasa/framework/api/authentication/internal/beans/TokenPayload.java

```diff
@@ -23,6 +23,9 @@ public class TokenPayload {
     @SerializedName("code")
     private String code;
 
```

Comment:
move this bean to the beans bundle so we can generate it easier

Reply:
Done